modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -741,6 +894,13 @@ private ObjectDefinition _publishObjectDefinition(
 			"pluralLabel", "portlet", "restContextPath",
 			"rootObjectDefinitionExternalReferenceCode", "scope", "status",
 			"storageType", "system", "titleObjectFieldName");
+	private static final List<String> _objectDefinitionObjectFieldNames =
+		Arrays.asList(
```

Comment:
These are (Test) Object Entry fields, but we want to test import of Object Definition, which has a different set of fields (see pic)
![Screenshot from 2023-09-22 09-08-07](https://github.com/liferay-headless/liferay-portal/assets/11851969/4603c1b9-d23d-4f0b-a930-e35e188f92cf)

Reply:
The only point of this variable is to make sure that the Object Definition imported has these Object Fields imported too

Reply:
Got it, I would also try to check if there is a match between import file field value and create ObjectDefinition field value, not just that field is present.
Admittedly this is a bit tricker than with Export flow, because backend is adding some generated values that are not present in the import file, but it should be possible.